demo/loaderUtilities.js

```diff
@@ -227,9 +232,26 @@ export async function getS3Files (s3, bucket, name) {
     const key = dir.split("/");
     table[key[2]] = list;
   }
-  return [filePaths, table];
+  return {
+    filePaths,
+    table
+  };
 }
 
 export function removeFileExtension (filename) {
   return filename.split('.').slice(0, -1).join('.');
 }
+
+function getLocalFiles () {
+  const dirs = {
+    '0_Preprocessed': [],
+    '1_Viz': [],
+    '2_Truth': [],
+    '3_Assessments': [],
+    '5_Schemas': [],
+    '6_Logs': [],
+    '7_Cals': []
```

Comment:
Don't need the quotes.

Reply:
I thought so too. Turns out, if you have a key that starts with a number followed by a non-numeric character, it generates a parsing error.